src/Api/IriConverterInterface.php

```diff
@@ -44,7 +44,7 @@ public function getItemFromIri(string $iri, bool $fetchData = false);
      *
      * @return string
      */
-    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH) : string;
+    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::CONFIG) : string;
```

Comment:
Why it is needed to add this new constant? IMO everything should be set in the Configuration and Extension classes.